Project.toml

```diff
@@ -64,6 +71,7 @@ TerminalLoggers = "0.1.4"
 julia = "1.6"
 
 [extras]
+KrylovKit = "0b1a1467-8014-51b9-945f-bf0ae24f4b77"
```

Comment:
Do you need both `[weakdeps]` and `[extras]`? Is that to be compatible with earlier Julia versions?

Reply:
If you don't include `[extras]`, you can't specify the `[compat]` entry without it complaining in earlier versions.